website/docs/reference/widgets/chart.mdx

```diff
@@ -124,11 +124,11 @@ Allows you to add data for Custom charts. You need to specify `type` for chart t
 }
 ```
 
-* **Type :** The type is used to define the chart type recognized by the FusionCharts. FusionCharts provides an extensive range of chart variants, with almost 100+ options to choose from. 
+* **Type :** The type is used to define the chart type recognized by the FusionCharts, written in JavaScript. FusionCharts provides an extensive range of chart variants, with almost 100+ options to choose from. For example: `"type": "column2d"`
```

Comment:
Here you have to mention that the JavaScript alias should be used, and for datasource the JSON format.
<img width="1046" alt="Screenshot 2023-07-21 at 12 10 04 PM" src="https://github.com/appsmithorg/appsmith-docs/assets/99252011/3ad8d80f-a4cf-4c5a-9477-a51c68e561e9">